source/draw/gpu/opengl/AndroidContext.ooc

```diff
@@ -27,19 +27,14 @@ AndroidContext: class extends OpenGLContext {
 		super()
 	}
 	createImage: override func (rasterImage: RasterImage) -> GpuImage {
-		result: GpuImage
-		version (optiGraphicbufferupload) {
-			if (rasterImage instanceOf(GraphicBufferYuv420Semiplanar)) {
-				graphicBufferImage := rasterImage as GraphicBufferYuv420Semiplanar
+		match(rasterImage) {
+			case (graphicBufferImage: GraphicBufferYuv420Semiplanar) =>
 				rgba := graphicBufferImage toRgba(this)
-				result = this _unpackRgbaToYuv420Semiplanar(rgba, rasterImage size, graphicBufferImage uvPadding % graphicBufferImage stride)
+				result := this _unpackRgbaToYuv420Semiplanar(rgba, rasterImage size, graphicBufferImage uvPadding % graphicBufferImage stride)
 				rgba referenceCount decrease()
-			}
-			else
-				result = super(rasterImage)
-		} else
-			result = super(rasterImage)
-		result
+				result
+			case => super(rasterImage)
+		}
```

Comment:
Why not simply always allow this? I don't think it improves readability by adding a `result` variable to be fair.